tests/test_formatter.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+import pytest
+import web3.web3.formatter as formatter
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    [
+    ('XE7338O073KYGTWWZN0F2WZ0R8PX5ZPPZS', '0x00c5496aee77c1ba1f0854206a26dda82a81d6d8'),
+    ('0x00c5496aee77c1ba1f0854206a26dda82a81d6d8', '0x00c5496aee77c1ba1f0854206a26dda82a81d6d8'),
+    ('00c5496aee77c1ba1f0854206a26dda82a81d6d8', '0x00c5496aee77c1ba1f0854206a26dda82a81d6d8'),
+    ('0x11f4d0a3c12e86b4b5f39b213f7e19d048276dae', '0x11f4d0a3c12e86b4b5f39b213f7e19d048276dae')
+    ]
+)
+def test_inputAddressFormatter(value, expected):
+    assert formatter.inputAddressFormatter(value) == expected
+
+@pytest.mark.parametrize(
+    "value",
+    [
+    ('0x0c5496aee77c1ba1f0854206a26dda82a81d6d8'),
+    ('0x0c5496aee77c1ba1f0854206a26dda82a81d6d8'),
+    ('00c5496aee77c1ba1f0854206a26dda82a81d6d'),
+    ('XE7338O073KYGTWWZN0F2WZ0R8PX5ZPPZE'),
+    ('0x'),
+    ]
+)
+def test_inputAddressFormatter2(value):
+    with pytest.raises(Exception):
```

Comment:
*nitpick*

It may be better to catch the specific exception class this raises to be sure this test isn't passing due to some other unexpected exception being raised.